module-1/README.md

```diff
@@ -69,67 +70,101 @@ In the terminal, change directory to the newly cloned repository directory:
 
 ```
-aws s3 mb s3://REPLACE_ME_BUCKET_NAME
+aws s3 mb s3://mythical-mysfits-bucket-name
 ```
 
-Now that we have created a bucket, we need to set some configuration options that enable the bucket to be used for [static website hosting](https://docs.aws.amazon.com/AmazonS3/latest/dev/WebsiteHosting.html).  This configuration enables the objects in the bucket to be requested using a registered public DNS name for the bucket, as well as direct site requests to the base path of the DNS name to a selected website homepage (index.html in most cases):
+#### Update the S3 Bucket Policy
+
+All buckets created in Amazon S3 are fully private by default.  In order to be used as a website accessible only through CloudFront, we need to create an S3 [Bucket Policy](https://docs.aws.amazon.com/AmazonS3/latest/dev/example-bucket-policies.html) that indicates objects stored within this new bucket may be accessed by a CloudFront origin access identity that we also create. Bucket policies are represented as JSON documents that define the S3 *Actions* (S3 API calls) that are allowed (or explicitly denied) to be performed by different *Principals* (in our case the CloudFront service).
+
+First we need to create a CloudFront Access Identity using the following CLI command.
 
 ```
-aws s3 website s3://REPLACE_ME_BUCKET_NAME --index-document index.html
+aws cloudfront create-cloud-front-origin-access-identity --cloud-front-origin-access-identity-config CallerReference=Mysfits,Comment=Mysfits
 ```
 
-#### Update the S3 Bucket Policy
+Record the **Id** from the output as we will need to update the bucket policy with this.
+
+```
+{
+    "CloudFrontOriginAccessIdentity": {
+        "CloudFrontOriginAccessIdentityConfig": {
+            "Comment": "Mysfits", 
+            "CallerReference": "Mysfits"
+        }, 
+        "S3CanonicalUserId": "ABCABCCCAfICCQD6m7oRw0uXOjANBgkqhkiG9w0BAQUFADCBiDELMAkGA1EXAMPLE", 
+        "Id": "ABCAJ2UCCR6DPCEXAMPLE"
+    }, 
+    "ETag": "ANPAJ2UCCR6DPCEXAMPLE", 
+    "Location": "https://cloudfront.amazonaws.com/2017-10-30/origin-access-identity/cloudfront/ANPAJ2UCCR6DPCEXAMPLE"
+}
+
+```
 
-All buckets created in Amazon S3 are fully private by default.  In order to be used as a public website, we need to create an S3 [Bucket Policy](https://docs.aws.amazon.com/AmazonS3/latest/dev/example-bucket-policies.html) that indicates objects stored within this new bucket may be publicly accessed by anyone. Bucket policies are represented as JSON documents that define the S3 *Actions* (S3 API calls) that are allowed (or not not allowed) to be performed by different *Principals* (in our case the public, or anyone).
+The JSON document for the necessary bucket policy is located at: `~/environment/aws-modern-application-workshop/module-1/aws-cli/website-bucket-policy.json`.  This file contains two strings that need to be replaced, the Id (indicated with `REPLACE_ME_ID`), and the bucket name you've chosen (indicated with `REPLACE_ME_BUCKET_NAME`).  
```

Comment:
Suggest this, the default privacy settings of S3 line is an important note I think, I'd like that to see remain.  Also, updated the REPLACE_ME_ID below to something more descriptive so that attendees still make the connection to what value they're inserting here.

"All buckets created in Amazon S3 are fully private by default.  In order for your CloudFront origin access identity to access the website content stored in your S3 bucket, we need to create and S3 [Bucket Policy](https://docs.aws.amazon.com/AmazonS3/latest/dev/example-bucket-policies.html) that indicates objects stored within this bucket may be accessed by only you CloudFront origin access ideneity.  The JSON document for the necessary bucket policy is located at: `~/environment/aws-modern-application-workshop/module-1/aws-cli/website-bucket-policy.json`.  This file contains two strings that need to be replaced, the Id (indicated with `REPLACE_ME_CLOUDFRONT_ORIGIN_ACCESS_IDENTITY_ID`), and the bucket name you've chosen (indicated with `REPLACE_ME_BUCKET_NAME`). "

Reply:
Line 86 still remains as buckets fully private by default. Replaced references to bucket with mythical-mysfits-bucket-name as per previous comment to maintain consistency.